src/applications/personalization/profile/components/personal-health-care-contacts/Contacts.jsx

```diff
@@ -56,8 +53,13 @@ const Contacts = ({ data }) => {
           <ul className="vads-u-margin-y--0">
             <li>Ask a staff member at your next appointment, or</li>
             <li>
-              Call us at <HelpDeskContact testId="help-desk" />. We’re here
-              Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.
+              Call your VA health facility.{' '}
+              <a
+                data-testid="find-locations-additional-info-link"
+                to="https://www.va.gov/find-locations/"
+              >
+                Find your health facility’s phone number
+              </a>
```

Comment:
In order for this anchor tag to function it needs to have a valid `href` attribute like this:

```html
<a
  data-testid="find-locations-additional-info-link"
  href="/find-locations/"
>
  Find your health facility’s phone number
</a>
```

I believe the [current recommendation from the design system team](https://design.va.gov/storybook/?path=/docs/components-va-link--default) is to use the <va-link> web component for all links instead of vanilla anchor tags so that the analytics events for clicks are tracked automatically like so:

``` html
  <va-link
    href="/find-locations/"
    text="Find your health facility’s phone number"
  />
  ```

Reply:
thank you